packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -1054,6 +1075,28 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
 		// Local data store does not have initial summary.
 		return {};
 	}
+
+	/**
+	 * A context should only be marked as deleted when its a remote context.
+	 * Session Expiry at the runtime level should have closed the container creating the local data store context
+	 * before delete is even possible. Session Expiry is at 30 days, and sweep is done 36+ days later from the time
+	 * it was unreferenced. Thus the sweeping container should have loaded from a snapshot and thus creating a remote
+	 * context.
+	 */
+	public delete() {
+		sendGCUnexpectedUsageEvent(
```

Comment:
Seems like we could just assert.  But I guess we'll get more info this way, so never mind.

Reply:
There's a missing todo here. Will add for GC Validation.